R/env.R

```diff
@@ -54,13 +54,13 @@
 #' env(fn)
 #'
 #'
-#' # new_env() creates by default an environment whose parent is the
```

Comment:
Yeah, but sometimes you just want a basic environment, not caring that it's a child of the empty environment. Maybe we could split the difference and have `new_env()` with no arguments as well as `child_env()`?